src/app/lib/config/services/amharic.js

```diff
@@ -136,31 +136,35 @@ const service = {
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'ስለ ውጪ ሊንኮች ያለን አቀራረብ፡፡',
+        text: 'ስለ ውጪ ሊንኮች ያለን አቀራረብ',
       },
       links: [
         {
           href: 'https://www.bbc.com/news/help-41670342',
           text: 'ቢቢሲን ለምን ማመን እንደሚገባዎ',
         },
         {
-          href: 'https://www.bbc.com/terms',
+          href: 'https://www.bbc.com//usingthebbc/terms/',
           text: 'የአጠቃቀም ደንብ',
         },
         {
-          href: 'https://www.bbc.com/privacy/',
+          href: 'https://www.bbc.com//usingthebbc/privacy/',
```

Comment:
Actually doesn't matter but looks a bit weird in the nav bar of the browser.
```suggestion
          href: 'https://www.bbc.com/usingthebbc/privacy/',
```